src/components/ui/DropdownMenuUsageExample.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+/**
+ * This file is used as an example for the Dropdown components.
+ * Once we're done with the example, we can delete this file.
+ */
+import { DeleteIcon, MenuKebabIcon, PenIcon } from '@/assets/images/icons';
+import { ContainerCount } from '@/features/groups/components/GroupCardList/ContainerCount';
+import { UserCount } from '@/features/groups/components/GroupCardList/UserCount';
+
+import {
+  Button,
+  Card,
+  DropdownMenu,
+  DropdownMenuButton,
+  DropdownMenuButtonIcon,
+  DropdownMenuButtonText,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  Icon,
+} from '.';
+
+export const DropdownMenuUsageExample = () => {
+  return (
+    <div className="p-4 flex flex-col gap-3">
+      {Array.from({ length: 10 }).map((_, i) => (
+        <DropDownMenuWithCardExample key={i} />
+      ))}
+    </div>
+  );
+};
+
+const DropDownMenuWithCardExample = () => {
+  const handleRenameClick = () => {
+    alert('Rename clicked!');
+  };
+
+  const handleDeleteClick = () => {
+    alert('Delete clicked!');
+  };
+  return (
+    <Card asChild>
```

Comment:
@kanta1207 
There are different approaches to styling this card. Would you like to do pair programming or have me improve it?